pallets/loans/src/lib.rs

```diff
@@ -1429,7 +1431,7 @@ impl<T: Config> Pallet<T> {
         })?;
 
         T::Assets::transfer(asset_id, &Self::account_id(), who, redeem_amount, false)
-            .map_err(|_| Error::<T>::InsufficientMarketLiquidity)?;
+            .map_err(|_| Error::<T>::InsufficientCash)?;
```

Comment:
do we still need `InsufficientMarketLiquidity`

Reply:
I think `InsufficientMarketLiquidity` should only appear in `ensure_liquidity`. Alternatively, We can add a new error for `PalletAccountBalanceLow` for this situation.